common/src/main/java/tachyon/LeaderSelectorClient.java

```diff
@@ -118,9 +118,9 @@ public void setCurrentMasterThread(Thread currentMasterThread) {
   }
 
   /**
-   * Start the Leader selection
+   * Start the Leader selection.
```

Comment:
Nit not caused by your change: If `Leader` is a class it should be `@link`ed to make this clear, otherwise it should be lowercased, and this class uses the wrong tense for its javadocs.